webpack.common.js

```diff
@@ -1,10 +1,9 @@
-const webpack = require('webpack')
+const webpack = require('webpack');
```

Comment:
no semi-colons please. I said: "Standard" standards. ....

Reply:
I have used https://standardjs.com/ will use it every time now, but it doesn't works with bracket styling properely.
Plz take a look to my comments
const { merge } = require('webpack-merge')
and
const myConfig = merge(
{
}
)

Are different ways of styling brackets jsbeutifyrc allows following options:
// [collapse|collapse-preserve-inline|expand|end-expand|none] Put braces on the same line as control statements (default), or put braces on own line (Allman / ANSI style), or put end braces on own line, or attempt to keep them where they are

Reply:
It allows to format HTML/CSS/JS https://github.com/beautify-web/js-beautify